qcodes/tests/test_channels.py

```diff
@@ -298,66 +307,95 @@ def test_loop_simple(self):
         data = loop.run()
         self.assertEqual(data.p1_set.ndarray.shape, (21, ))
         self.assertEqual(len(data.arrays), 7)
         for chan in ['A', 'B', 'C', 'D', 'E', 'F']:
-            self.assertEqual(getattr(data, 'testchanneldummy_Chan{}_temperature'.format(chan)).ndarray.shape, (21,))
+            self.assertEqual(getattr(data, 'testchanneldummy_Chan{}_temperature'
+                                     .format(chan)).ndarray.shape, (21,))
 
     def test_loop_measure_channels_individually(self):
-        p1 = Parameter(name='p1', vals=Numbers(-10, 10), get_cmd=None, set_cmd=None)
-        loop = Loop(p1.sweep(-10, 10, 1), 1e-6).each(self.instrument.channels[0].temperature,
-                                                     self.instrument.channels[1].temperature,
-                                                     self.instrument.channels[2].temperature,
-                                                     self.instrument.channels[3].temperature)
+        p1 = Parameter(name='p1', vals=Numbers(-10, 10), get_cmd=None,
+                       set_cmd=None)
+        loop = Loop(p1.sweep(-10, 10, 1), 1e-6).each(self.instrument.
+                                                     channels[0].temperature,
+                                                     self.instrument.
+                                                     channels[1].temperature,
+                                                     self.instrument.
+                                                     channels[2].temperature,
+                                                     self.instrument.
+                                                     channels[3].temperature)
         data = loop.run()
         self.assertEqual(data.p1_set.ndarray.shape, (21, ))
         for chan in ['A', 'B', 'C', 'D']:
-            self.assertEqual(getattr(data, 'testchanneldummy_Chan{}_temperature'.format(chan)).ndarray.shape, (21,))
+            self.assertEqual(getattr(data, 'testchanneldummy_Chan{}_temperature'
+                                     .format(chan)).ndarray.shape, (21,))
 
     @given(values=hst.lists(hst.floats(0, 300), min_size=4, max_size=4))
     @settings(max_examples=10, deadline=None)
     def test_loop_measure_channels_by_name(self, values):
-        p1 = Parameter(name='p1', vals=Numbers(-10, 10), get_cmd=None, set_cmd=None)
+        p1 = Parameter(name='p1', vals=Numbers(-10, 10), get_cmd=None,
+                       set_cmd=None)
         for i in range(4):
             self.instrument.channels[i].temperature(values[i])
-        loop = Loop(p1.sweep(-10, 10, 1), 1e-6).each(self.instrument.A.temperature,
-                                                     self.instrument.B.temperature,
-                                                     self.instrument.C.temperature,
-                                                     self.instrument.D.temperature)
-        data = loop.run()
+        loc_fmt = '{date}/#{counter}_{name}_{date}_{time}'
```

Comment:
Lets make this `data/{date}/#{counter}_{name}_{date}_{time}` to match how it was before

Reply:
Is this change apply to everyone of them or only this line?